scripts/generate_teams.py

```diff
@@ -30,6 +30,8 @@
     "Afgr1087",
     "RuiheLi",
     "smitasahu2",
+    "agriyakhetarpal",
+    "arjxn-py",
```

Comment:
I hope this will automatically update the HTML files on the next scheduled run.

Reply:
```suggestion
    "agriyakhetarpal",
    "arjxn-py",
    "kratman"
```

Should we add @kratman as well here?

Reply:
I am fine with being added here. I got busy with work, but I plan on continuing my contributions long term